processor/tailsamplingprocessor/config.go

```diff
@@ -96,6 +96,8 @@ type StringAttributeCfg struct {
 	// from the regular expressions defined in Values.
 	// CacheMaxSize will not be used if EnabledRegexMatching is set to false.
 	CacheMaxSize int `mapstructure:"cache_max_size"`
+	// InvertMatch indicates whether values should (false) or should not (true) match against attribute values.
```

Comment:
I feel like this could use an example. Like: "Example, when the key 'tenant' is set to 'acme' and this option is enabled, traces containing spans for the tenant 'ecorp' will match, while traces with spans for 'acme' will not."